tests/Models/Animal.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests\Models;
+
+use MongoDB\Laravel\Eloquent\Model as Eloquent;
+use MongoDB\Laravel\Query\Builder;
+
+/**
+ * @property string $name
+ * @property string $country
+ * @property bool $can_be_eaten
+ * @mixin Eloquent
+ * @method static Builder create(...$values)
+ * @method static Builder truncate()
+ * @method static Eloquent sole(...$parameters)
+ */
+final class Animal extends Eloquent
```

Comment:
I don't see what is tested with this model class. Hidden fields and the `toArray` feature are tested with `HiddenAnimal`.

Reply:
The idea of the two classes is that one has the hidden property and another doesn't. As to prove the working of the hidden property.

Or do you suggest to have one class and use different fields to test?

Reply:
IMHO, the test on HiddenAnimal already covers the feature.

Reply:
Fair enough, will remove this class and its usage